docs/markdown/how-to.md

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
let's update these file names to match the page file names (and therefore routes) exactly